web-frontend/src/const/assets/assets.ts

```diff
@@ -1,17 +1,5 @@
 export const XMA = 'XMA';
 
-export interface ITokenData {
```

Comment:
I believe this interface is necessary no ? because it's what the component receives

Reply:
As far as I saw it was a duplicate of IToken (now named Asset)

Reply:
it's a fake duplicate, it takes some elements from Asset interface and adds a bunch of things that the component receives, but it's also defined in the component, not sure why it's here twice